logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/message/MessageEnvelopeCreator.kt

```diff
@@ -42,7 +43,8 @@ interface MessageEnvelopeCreator {
 
 class MessageEnvelopeCreatorImpl(
     private val proteusClientProvider: ProteusClientProvider,
-    private val protoContentMapper: ProtoContentMapper = MapperProvider.protoContentMapper(),
+    private val selfUserId: UserId,
+    private val protoContentMapper: ProtoContentMapper = MapperProvider.protoContentMapper(selfUserId = selfUserId),
```

Comment:
We are injecting `ProtoContentMapper` so I think we can simplify this by removing the default value. I guess same thing applies for other classes as well.